### ISSUE TITLE
Populate the card description with content of Obsidian document

### ISSUE THREAD

**Issue by noescapeartist** [OfficerHalf/obsidian-trello#41]
I would love to be able to populate the Trello card description with the content of my Obsidian document.

**Comment by OfficerHalf** [comment]
oh, interesting. I guess you could sort of use obsidian as an editor for trello at that point. 🤔 This plugin isn't super high on my list of things right now, but I think this would be a nice QOL change.

**Comment by bdalia** [comment]
Hi, 
i was writing the same request but i'm happy to see that is already submitted.
 
Obsidian editor is much better than the one provided inside trello. For example, with obsidian i can easly mix text and images during the writing of a note (by simply pasting a copied image at the cursor position. This is also possible in trello, but is a bit tricky and require additional manual steps. 

The plug-in may do a small conversion, by uploading attachements included in the note and linking them with the md sintax expected by trello. 

This would by a huge time saver for who need to create card that must show mixed text and images.

Regards.

**Comment by ooker777** [comment]
Hi, I'm willing to work on this feature. While I'm definitely not an expert, I have little knowledge in Obsidian plugin development (see my previous plugin [Graphvidian](https://github.com/QuaCau-TheSphere/Graphvidian)), so your support wouldn't be too much. Do you have some free time to help me understand the plugin so that I can contribute? Many thanks

**Comment by bdalia** [comment]
Hi @ooker777, if your reply is for me, i cannot help because i'm not developing, i only can contribute for plug-in improvement by giving my feedback during the use with Trello.
I don't know if @OfficerHalf (the plug-in owner) can give you some advice on it. 

Regards.

**Comment by OfficerHalf** [comment]
> Hi, I'm willing to work on this feature. While I'm definitely not an expert, I have little knowledge in Obsidian plugin development (see my previous plugin [Graphvidian](https://github.com/QuaCau-TheSphere/Graphvidian)), so your support wouldn't be too much. Do you have some free time to help me understand the plugin so that I can contribute? Many thanks

Yeah certainly you can DM me on discord, I'm on the Obsidian server.

**Comment by ChrisChinchilla** [comment]
I have started this…
